### ISSUE TITLE
OSX Chrome 55.0.2883.95 Blocking non-store extensions completely without dev mode

### ISSUE THREAD

**Issue by mikejk8s** [dhowe/AdNauseam#703]
As of today AdNauseam was automatically uninstalled from my 55.0.2883.95 Chrome 64 on OSX and I'm unable to reinstall it using the normal method, I had to uninstall it and enable developer mode. Just letting you guys know in case you weren't aware and wanted to update the docs/video.

Thanks!

**Comment by mikejk8s** [comment]
Actually, after further testing it it's uninstalling it everytime Chrome opens now.

**Comment by dhowe** [comment]
Thanks @mikejk8s, that is much appreciated -- we are aware and preparing a statement and some workaround info to be released in a few hours (though, as you probably know, there is not much the average user can do in this case)...

**Comment by timthelion** [comment]
Well, this issue certainly got me to install adnauseam!
You can now use the sales mark "An adblocker so effective Google banned it!" (SM)

Any idea if Chromium will also block Adnauseam?

**Comment by Ajedi32** [comment]
A couple questions:

1. Are you saying Chrome is uninstalling the extension even if you build it yourself and install it in dev mode? That's strange, because I thought Chrome only _notified_ users when they open Chrome with a dev mode extension installed. I didn't think it automatically uninstalled them.
2. Does Chrome now automatically uninstall all dev mode extensions on OS X upon restarting the browser, or just this one?
3. Does the problem persist if you change the extension name in the manifest before installing it?

**Comment by mikejk8s** [comment]
@Ajedi32 

I won't really have time to test any of this until this weekend. I'll respond with what I did and what I ran into. I'm actually on my Windows machine right now and it hasn't uninstalled at all. I'm on 55.0.2883.87 m here. If this is hindering development I can try to test it tomorrow, but to be completely honest I'm unfamiliar with Chrome's extension system beyond enabling dev-mode and installing the one offs on occassion. 

At some random point during the day my Chrome on OSX popped up that it had uninstalled the extension as it wasn't in the Chrome web store. I'd used the extension for about a month and had no issues.

To answer your questions;

1. I didn't try building/packing the extension from source and injecting it. Just the normal method of downloading it, going into chrome://extensions and dragging it into the UI. It'd yell at me that it's not in the store, so I'd restart the browser, enable Dev mode, then drag it with no problems until the browser restarted then it would immediately uninstall it.

2. I don't believe I have any other dev mode extensions. If you want to post one I can try it.

3. Haven't done so. I can try this weekend.
